tron/commands/cmd_utils.py

```diff
@@ -40,6 +41,15 @@ class ExitCode(object):
 opener = open
 
 
+def get_default_server():
+    return DEFAULT_CONFIG['server']
+
+
+def tron_jobs_completer(prefix, parsed_args, **kwargs):
+    default_client = Client(get_default_server())
```

Comment:
How safe is this when it doesn't work? Do users get spammed with stacktraces as tab-complete suggestions?

Reply:
Nope, you only see those stack traces if you have that "_ARC_DEBUG" env var set. Otherwise, nothing happens (you just get no suggestions on pressing tab)